packages/calypso-color-schemes/src/shared/color-schemes/_blue.scss

```diff
@@ -144,6 +144,12 @@ Used studio-blue for the primary+accent.
 	--color-sidebar-submenu-hover-background: transparent;
 	--color-sidebar-submenu-hover-text: var(--theme-text-color);
 
+	/* Sidebar Submenu - Nav Redesign */
+	--color-navredesign-sidebar-submenu-text: #e2ecf1; /* $menu-submenu-text: */
+	--color-navredesign-sidebar-menu-selected-text: var(--theme-text-color);
+	--color-navredesign-sidebar-submenu-selected-text: var(--theme-text-color);
+	--color-navredesign-sidebar-submenu-hover-text: var(--theme-text-color);
```

Comment:
![badge](https://img.shields.io/badge/review-imo-orange.svg)

The files located at `packages/calypso-color-schemes/src/shared/color-schemes/_*.scss` were initially added for Nav Unification and are tailored specifically to it, incorporating Calypso-specific color designs. However, given that Nav Redesign fundamentally diverges from Nav Unification, inconsistencies have arisen, particularly with the variable names used in the color schemes.

Therefore, in the future, I suggest that we import all Core color schemes as faithfully as possible to their original specifications. This includes maintaining the original variable names and color palette designs from Core. We better minimize using Calypso-specific names and designs to align our project more closely with Core standards.